tests/cluster_policies/__init__.py

```diff
@@ -27,6 +27,9 @@
 
 # [START example_cluster_policy_rule]
 def task_must_have_owners(task: BaseOperator):
+    if task.owner and not isinstance(task.owner, str):
+        raise AirflowClusterPolicyViolation(f'''owner should be a string. Current value: {task.owner}''')
```

Comment:
```suggestion
        raise AirflowClusterPolicyViolation(f'''owner should be a string. Current value: {task.owner!r}''')
```

Reply:
In some cases, `str()` returns the output, which might look like a valid string when it really is a complex object, so we should use `repr()` to avoid confusion.
```
>>> f"Value: {datetime.datetime.now()}"
'Value: 2022-04-29 19:19:05.618692'
>>> f"Value: {datetime.datetime.now()!r}"
'Value: datetime.datetime(2022, 4, 29, 19, 19, 8, 385189)'
```

Reply:
Thanks, used repr value.